grails-app/services/org/pih/warehouse/core/DocumentTemplateService.groovy

```diff
@@ -114,7 +114,7 @@ class DocumentTemplateService {
                     expectedShippingDate: orderItem?.estimatedReadyDate ? orderItem?.estimatedReadyDate?.format("MMM dd, yyyy") : ""
             ]
         }
-        def orderAdjustments = orderInstance?.orderAdjustments?.findAll { !it.canceled }?.collect { OrderAdjustment orderAdjustment ->
+        def orderAdjustments = orderInstance?.orderAdjustments?.findAll { !it.canceled }?.sort()?.collect { OrderAdjustment orderAdjustment ->
```

Comment:
This feels like a case where the default sort order of the association would make sense.

Either in the domain class
```
class OrderAdjustment {
    static mapping = { 
        sort dateCreated : "desc"
    }
}
```
Or in the association on Order
```
class Order { 
    static mapping = {
        orderAdjustments(sort:'dateCreated', order:'desc')
    }
}
```

Reply:
I followed your solution with `compareTo` with fallbacks